cypress/e2e/experiments.cy.ts

```diff
@@ -96,6 +96,7 @@ describe('Experiments', () => {
         cy.get('[data-attr="experiment-creation-date"]').contains('a few seconds ago').should('be.visible')
         cy.get('[data-attr="experiment-start-date"]').should('not.exist')
 
+        cy.wait(1000)
```

Comment:
Some re-rendering glitch causes the click to fail. Waiting for a bit solves this; no issues when testing manually.